boards/opentitan/earlgrey-cw310/.cargo/config.toml

```diff
@@ -2,5 +2,20 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 # Copyright Tock Contributors 2022.
 
+include = [
+  "../../../cargo/tock_flags.toml",
+  "../../../cargo/unstable_flags.toml",
+  "../../../cargo/riscv_flags.toml",
+]
+
+[build]
+target = "riscv32imc-unknown-none-elf"
+rustflags = [
+  "-Z", "virtual-function-elimination"
+]
+
```

Comment:
Okay, maybe I'm being obnoxious, but this is exactly what we _don't_ want I think. Letting one-offs / "this magic line is necessary for this board to work" slip into the hidden config file. I feel like we should add a `cargo/earlgrey-cw310.toml` or similar somewhere and include it to pull this in.

Reply:
Hmm, need to look into this and what happened here. That line should have already been there (aka not be green).

Reply:
Fixed.

Reply:
As explained [here](https://github.com/tock/tock/pull/4054#issuecomment-2204016639) in the original PR, this is not "magically need it for this board to work," this is already in the Makefiles as a "magic" `VFUNC` variable that is only enabled for the earlgrey board.

This is a code size optimization that is still considered unstable and dangerous (because I guess it is not always correct) but saves a bunch of code size.

Reply:
In particular, I'd argue this is significantly _more_ hidden and obscure in the Makefiles where it is abstracted behind some custom variable name you have to chase down (basically via the Makefile equivalent of `#ifdef`s).

Reply:
I define "hidden" as if I `grep` for `virtual-function-elimination` will it show up. Living in a hidden file / directory, the answer is no; living in a Makefile or anything not hidden, the answer is yes. The `.` of `.cargo` is a cancer.